unicodetools/src/main/resources/org/unicode/props/ExtraPropertyValueAliases.txt

```diff
@@ -205,6 +204,11 @@ Do_Not_Emit_Type ; Preferred_Spelling       ; Preferred_Spelling
 # @missing: 0000..10FFFF; kSrc_NushuDuben ; <none>
 # @missing: 0000..10FFFF; kReading ; <none>
 
+# @missing: 0000..10FFFF; kEH_Core  ;  N
+kEH_Core ; C ; C
+kEH_Core ; L ; L
+kEH_Core ; N ; N
```

Comment:
should we set the long names to Core / Legacy / None?
https://www.unicode.org/reports/tr57/tr57-2.html#N101BA

Reply:
Hm. That wording seems a bit squishy (I read it as explaining the origin of the one-letter values rather than formally defining long names), but since this is all provisional and since defining long names here will have no effect on the Standard and associated published data files, and indeed no effect anywhere except in display in the online tools, we might as well do it for readability.